src/components/rangeUsePlanPage/plantCommunities/PlantCommunityBox.js

```diff
@@ -47,7 +48,8 @@ const PlantCommunityBox = ({
     approved,
     notes,
     communityType,
-    monitoringAreas
+    monitoringAreas,
+    id
```

Comment:
@calebissharp I'm wondering if this is the id in our error

Reply:
@micheal-w-wells The error isn't client-sided, so this most likely isn't the cause. It's going to be something introduced in the versioning changes in the API.